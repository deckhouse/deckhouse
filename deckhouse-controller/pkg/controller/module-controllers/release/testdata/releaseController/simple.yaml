---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleSource
metadata:
  annotations:
    modules.deckhouse.io/registry-spec-checksum: 38557e472e4e2bd8695fc58a255ec3dd
  finalizers:
    - modules.deckhouse.io/release-exists
  name: foxtrot
spec:
  registry:
    ca: ""
    dockerCfg: YXNiCg==
    repo: dev-registry.deckhouse.io/team/foxtrot/modules
    scheme: HTTPS
status:
  message: ""
  moduleErrors: []
  modules:
    - name: mcplay
      policy: foxtrot-alpha
    - name: parca
      policy: foxtrot-alpha
  modulesCount: 2
  syncTime: "2024-05-03T21:05:05Z"
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleUpdatePolicy
metadata:
  name: foxtrot-alpha
spec:
  moduleReleaseSelector:
    labelSelector:
      matchLabels:
        source: foxtrot
  releaseChannel: Alpha
  update:
    mode: Auto
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  labels:
    module: parca
    modules.deckhouse.io/update-policy: foxtrot-alpha
    release-checksum: 98d00f741c99e06e6c6c4d18b763c550
    source: foxtrot
  name: parca-v1.4.3
  ownerReferences:
    - apiVersion: deckhouse.io/v1alpha1
      controller: true
      kind: ModuleSource
      name: foxtrot
      uid: 71d2300f-700b-452a-896a-6a3805f9cef7
spec:
  changelog:
    features:
      - Bump parca version
  moduleName: parca
  version: 1.4.3
  weight: 900
status:
  approved: false
  message: ""
  phase: Pending
  pullDuration: 0s
  size: 0
  transitionTime: "2024-05-03T20:55:49Z"
