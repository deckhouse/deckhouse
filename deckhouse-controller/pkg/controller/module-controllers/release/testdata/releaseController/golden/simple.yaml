---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleSource
metadata:
  annotations:
    modules.deckhouse.io/registry-spec-checksum: 38557e472e4e2bd8695fc58a255ec3dd
  creationTimestamp: null
  finalizers:
  - modules.deckhouse.io/release-exists
  name: foxtrot
  resourceVersion: "999"
spec:
  registry:
    ca: ""
    dockerCfg: YXNiCg==
    repo: dev-registry.deckhouse.io/team/foxtrot/modules
    scheme: HTTPS
  releaseChannel: ""
status:
  message: ""
  moduleErrors: []
  modules:
  - name: mcplay
    policy: foxtrot-alpha
  - name: parca
    policy: foxtrot-alpha
  modulesCount: 2
  syncTime: "2024-05-03T21:05:05Z"
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  annotations:
    release.deckhouse.io/notified: "false"
  creationTimestamp: null
  labels:
    module: parca
    modules.deckhouse.io/update-policy: foxtrot-alpha
    release-checksum: 98d00f741c99e06e6c6c4d18b763c550
    source: foxtrot
  name: parca-v1.4.3
  ownerReferences:
  - apiVersion: deckhouse.io/v1alpha1
    controller: true
    kind: ModuleSource
    name: foxtrot
    uid: 71d2300f-700b-452a-896a-6a3805f9cef7
  resourceVersion: "1002"
spec:
  changelog:
    features:
    - Bump parca version
  moduleName: parca
  version: 1.4.3
  weight: 900
status:
  approved: false
  message: ""
  phase: Deployed
  pullDuration: 9m15s
  size: 3
  transitionTime: "2019-10-17T15:33:00Z"
